src/components/ui/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import { HTMLAttributes, useId } from 'react'
+
+type InputProps = HTMLAttributes<HTMLInputElement> & {
+  label?: string,
+  classNameInput?: string,
+  classNameLabel?: string
+}
+
+export default function Input({
+  label,
+  classNameInput,
+  classNameLabel,
+  ...rest }: InputProps) {
+  const id = useId()
+
+  const classNameInputMerge = "h-[54px] bg-secondary border border-gray-300 text-gray-900 text-L rounded-lg focus:ring-brand-solid-hover block w-full p-2.5 "
```

Comment:
Your are using "text-sm"  , "text-L" It's a good idea to use consistent naming conventions for better readability and maintainability.